keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/jackson/KeelEc2ApiModule.kt

```diff
@@ -51,14 +51,28 @@ import com.netflix.spinnaker.keel.ec2.jackson.mixins.StepAdjustmentMixin
 import com.netflix.spinnaker.keel.ec2.jackson.mixins.StepScalingPolicyMixin
 import com.netflix.spinnaker.keel.ec2.jackson.mixins.TargetGroupAttributesMixin
 import com.netflix.spinnaker.keel.ec2.jackson.mixins.TargetTrackingPolicyMixin
+import com.netflix.spinnaker.keel.jackson.SerializationExtensionRegistry
 
-fun ObjectMapper.registerKeelEc2ApiModule(): ObjectMapper = registerModule(KeelEc2ApiModule)
+fun ObjectMapper.registerKeelEc2ApiModule(serializationExtensionRegistry: SerializationExtensionRegistry): ObjectMapper {
+  with(serializationExtensionRegistry) {
```

Comment:
This is where the default serializers and deserializers are now registered. By the time we get here, the Netflix components will have already have initialized and had a chance to add customizations to the registry.